### PR TITLE
AC-579: Add Privacy Policy link to the app's Settings page

### PR DESCRIPTION
**Pull request by deepak140596**
## Description of what I changed

Added a link to Privacy Policy to Settings page of the app. 
On click, it opens the default browser to view the page.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-579

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsRecyclerViewAdapter.java [hunk 1]
```diff
@@ -53,12 +54,21 @@ public void onBindViewHolder(@NonNull SettingsViewHolder holder, int position) {
...
         if (position==0){
             holder.mRowLayout.setOnClickListener(new View.OnClickListener() {
-                    @Override
-                    public void onClick(View v) {
-                        Intent i = new Intent(v.getContext() , LogsActivity.class);
-                        v.getContext().startActivity(i);
-                        }
-                });
+                @Override
+                public void onClick(View v) {
```
Comment (VANKINEENITAWRUN) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsRecyclerViewAdapter.java]: @deepak140596  Can you use a lambda for this?

Comment (deepak140596) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsRecyclerViewAdapter.java]: Thanks @VANKINEENITAWRUN , changed the lines to lambda form.

**Comment by sanjulamadurapperuma** [comment]
Nice work @deepak140596 All looks good :)

**Comment by f4ww4z** [comment]
@deepak140596 Did you see the merge conflict? Just pull in the latest changes from master, I've merged a few other PRs.

**Comment by VANKINEENITAWRUN** [comment]
@deepak140596 first fast-forward the local openmrs master branch then rebase your branch onto it and solve merge conflicts. then make PR.

**Comment by deepak140596** [comment]
Thnaks @VANKINEENITAWRUN, doing the changes.

**Pull request closed without merging**